### DISCUSSION TITLE
加一个刮削+改名+整理电影/电视剧的版本

### DISCUSSION THREAD

**Discussion by qiqijy** [Yuukiy/JavSP#305] [想法 | Ideas]
这个刮削、整理的功能我觉得非常好用，十分便于整理命名比较混乱的媒体源文件
可以增加豆瓣、IMDb、TheMovieDb、TheTVDB等电影、电视剧的刮削站点么，以支持常规媒体文件的刮削整理
电视剧涉及分季等逻辑可能改动较大，但是电影这块逻辑应该相差不大吧
或者简化成刮削>规则化改名>整理文件夹即可，后续信息刮削可以用现有的刮削工具完成

**Comment by glyh** [comment]
我个人觉得超出这个项目范围了。两种解决方案：
1. 在另一个Github项目里重做类似功能
2. 把本项目拆成若干个组件，用户按需取用

**Comment by qicfan** [comment]
你需要的是TinyMediaManger
